AdminApi.Sdk/Endpoints/Identities/Types/Requests/CreateIdentityRequest.cs

```diff
@@ -7,6 +7,7 @@ public class CreateIdentityRequest
     public required string ClientId { get; set; }
     public required byte[] IdentityPublicKey { get; set; }
     public required string DevicePassword { get; set; }
+    public string? CommunicationLanguage { get; set; }
```

Comment:
We don't need the backwards compatibility for the Admin API, as we're the only ones that use it.